docs/guides/flavors/android.md

```diff
@@ -85,23 +85,6 @@ flavors:
 
 Since the project contains flavors, `shorebird init` generates an app per flavor.
 
-:::tip
```

Comment:
I’d recommend mentioning you can view your apps from the console instead of removing the tip entirely

Reply:
totally agree! 🙌🏻